payments-core/src/main/java/com/stripe/android/payments/bankaccount/navigation/CollectBankAccountResult.kt

```diff
@@ -28,33 +28,37 @@ sealed class CollectBankAccountResult : Parcelable {
 
 @Parcelize
 data class CollectBankAccountResponse(
-    val intent: StripeIntent,
+    /**
+     * The payment / setup intent. Expect null for deferred payments.
+     */
+    val intent: StripeIntent?,
```

Comment:
This should have been nullable before. Currently, `CollectBankAccountLauncher.presentWithDeferredPayment` would just fail because it requires the intent.

(iOS code for more reference: https://github.com/stripe/stripe-ios/blob/a94cc164cb87cc00730b5d3a5aaaf16b40c1c07f/StripePayments/StripePayments/Source/Helpers/STPBankAccountCollector.swift#L533)